07-html.Rmd

```diff
@@ -408,3 +408,60 @@ In Section \@ref(latex-hardcore), we mentioned that if you feel the constraint o
+
+If you want to ensure that each browser or platform on which your document is viewed uses a version of your icon with optimal resolution for its specific layout, you may use a service such as https://realfavicongenerator.net/ to generate a set of favicons and a slightly more complex version of the header HTML code. This service is currently used by the **pkgdown** package's `pkgdown::build_favicon()` function to make a set of favicons out of R package logos. 
+
+
+## Sharing HTML content on the web
+
+One appealing aspect of rendering R Markdown content to HTML files is that it is very easy to host these files on the internet and share them just as one shares any other website. This section briefly summarizes numerous options for sharing the HTML documents which you create. 
+
+### R-specific services
+
+RStudio PBC offers a number of services for publishing various types of content created in R Markdown to the internet. These services make it particularly easy to publish content by using the RStudio IDE or the **rsconnect** package.
+
+- [**RPubs**](https://rpubs.com/) enables free hosting of static single-file R Markdown content. It is easy to publish via the Publish button in the RStudio IDE or the `rsconnect::rpubsUpload()` function. Please see the "Getting Started" page (https://rpubs.com/about/getting-started) for more details.
+
+- [**shinyapps.io**](https://www.shinyapps.io/) allows for hosting dynamic content which requires a server running R. For example, one can host interactive R Markdown documents which use the `runtime: shiny` option to include Shiny components. shinyapps.io is an analog to RPubs for Shiny applications. Apps and interactive R Markdowns can be published using the push-button in the RStudio IDE or the `rsconnect::deployApp()` function. See the user guide (https://docs.rstudio.com/shinyapps.io/index.html) for more details. 
+
+- [**bookdown.org**](https://bookdown.org/home/about/) offers free hosting specifically for books written with the **bookdown** package. You may easily publish static output files of your book using the `bookdown::publish_book()` function. This platform is based on RStudio PBC's enterprise RStudio Connect product (see below).
+
+- [**RStudio Connect**](https://rstudio.com/products/connect/) is an enterprise product which organizations may run on their own servers. It can host a wide variety of content created in R (R Markdown documents, Shiny apps, APIs) in a secured environment with document-level access controls, viewership history, and more. Similar to the open-source RPubs and shinyapps.io offerings, content can be published to RStudio Connect using manual upload, the **rsconnect** package, or with Git-based deployment. 
+
+### Static website services
+
+In addition to R-specific services, you may host your HTML document on many freely available static website hosting services. These include:
```

Comment:
Maybe we should add a sentence about static website ? 

Example to be modified but gives the idea of what I have in mind: 

> In a few words, a simple static website is composed of an `index.html` (the main page), some other HTML, JavaScript, CSS files, and also some images or other content for the website.  Such folder can be hosted as-is on web servers to be rendered in usual browser.  When Rmarkdown is used to render to HTML output format, it results in a static website form, the minimal one being the standalone document where everything is included in a single HTML document, and more complex one being **blogdown** output where a static website generator is used. See [Section 2.1](https://bookdown.org/yihui/blogdown/static-sites.html) in [@blogdown2017].

As a result, in addition to R-specific services, you may host your HTML document on many freely available static website hosting services. On the R community, the ones mainly used are:

Reply:
Thank you -- this is great! I like the idea of including it instead of just linking it. As you point out, it does seem very helpful for anyone uploading their document to have the right mental model in mind. Especially for those reading the print version, they are probably a lot more likely to read text in the body than in other links.